### ISSUE TITLE
N00b request

### ISSUE THREAD

**Issue by LordSexy** [vintlabs/fauxmoESP#174]
Hey guys. 
Sorry for my lack of knowledge. Im a hobbyist.
Like I said, I'm a noob, but I dont know where I facked up at any time.
I can see the cheap ass ESP8266 (D1 mini) on my router. The ESP8266 even spits out data in the serial monitor. But I cant add anything when searching with the alexa app for android. 

Please tell me which information I can provide so you might get an idea about my problem. For sure the text above is not much....
Thanx for your understanding. 

FauxmoESP Version 3.3.0
ESPAsyncTCP-master fresh form github (me-no-dev)
Arduino IDE 1.8.13

**Comment by pvint** [comment]
Sorry for the slow reply. First thing I would check is that they are both on the same WiFi network (ie: is your Android on the same 2.4GHz wifi as the ESP?)

**Comment by LordSexy** [comment]
It was in the 5 GHz network, but I changed that and tried again. Like I said the communication with the router works. Just not the communication with Alexa

**Comment by LordSexy** [comment]
Yeah. F*** me. I got a ESP32 from a friend, flashed it and it worked. Now my ESP8266 also work. Same procedure as before, same code. I dont know why it works, but hey...
THX

**Comment by LordSexy** [comment]
EDIT: ESP32 working "flawless". I can add 13 instances with one ESP32. I need 16 so Ill guess I have to add 2 ESP32 and a few lines for SoftwareSerial.

But I cant get the ESP8266 working.
Ill guess I have a not compatible 8266 module (like I said. Sheep ass german amazon destributor).
How can I get the information which core im using.

Platform | Gen1 | Gen2 | Gen3
-- | -- | -- | --
ESP8266 Core 2.3.X | OK | OK? | OK (1)
ESP8266 Core 2.4.0 | OK (2) | OK? (2) | OK (1, 2)
ESP8266 Core 2.4.1 | OK (2) | OK? (2) | OK (1, 2)
ESP8266 Core 2.4.2 | OK (2) | OK? (2) | OK (1, 2)
ESP32 | OK | OK? | OK (1)

And I found the explanation for note (1) but where ist the condition for note (2). 
Thank you for providing this software and even maintaining it. 
And Im glad that you even look into the issue thread in a reasonable time. Ive seen repos with 1 year worth of unanswered problems.

**Comment by Tejas-MD** [comment]
@pvint you can close this.
